neurons/rewarder_test.py

```diff
@@ -73,10 +72,10 @@ def test_update_scores_multiple_competitors_no_reduction():
 def test_update_scores_multiple_competitors_with_some_reduced_shares():
     # Set up initial data for multiple competitors
     competition_leaders = {
-        "competition1": CompetitionLeader(hotkey="competitor1", leader_since=datetime.now() - timedelta(days=14 + 3 * 7)),
-        "competition2": CompetitionLeader(hotkey="competitor2", leader_since=datetime.now() - timedelta(days=14 + 6 * 7)),
-        "competition3": CompetitionLeader(hotkey="competitor3", leader_since=datetime.now() - timedelta(days=14)),
-        "competition4": CompetitionLeader(hotkey="competitor4", leader_since=datetime.now() - timedelta(days=14)),
+        "competition1": CompetitionLeader(hotkey="competitor1", leader_since=datetime.now() - timedelta(days=30 + 3 * 7)),
```

Comment:
you can change "competitior1" for a name "competitior_30_percent_reduction"

Reply:
etc

Reply:
this is mobbing